app/models/course.rb

```diff
@@ -606,7 +610,7 @@ def is_running?
   end
 
   def is_withdrawn?
-    content_status == "withdrawn" || not_running?
+    content_status == "withdrawn" || content_status == :withdrawn || not_running?
```

Comment:
```suggestion
    content_status.match?(/withdrawn/)  || not_running?
```

why is there symbol issue?

Reply:
For some reason there was an inconsistency with how we populated the `content_status` field - in the factory we set it to a symbol, but in the app we seem to set it as a string - rather than try and line everything up, I've put this instead. Probably needs a refactor though!